src/index.html

```diff
@@ -9,6 +9,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+          src="/images/logo.png" alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__items">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__items">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
```

Comment:
fix the formatting everywhere
```suggestion
        <ul class="nav__list">
          <li class="nav__items">
            <a href="#" class="nav__link is-active">
              Apple
            </a>
          </li>
          
          <li class="nav__items">
            <a href="#" class="nav__link">
              Samsung
            </a>
          </li>
```